app/templates/components/event-card.hbs

```diff
@@ -23,31 +23,10 @@
       </SmartOverflow>
       <div class="meta mb-2">
         <span class="date">
-          {{#if this.event.isSingleDay}}
-            {{general-date this.event.startsAt 'date-time-long' tz=this.event.timezone}}
-            - 
-            {{general-date this.event.endsAt 'time-tz-short' tz=this.event.timezone}}
-          {{else}}
-            {{general-date this.event.startsAt 'date-time-long' tz=this.event.timezone}}
-            -
-            {{general-date this.event.endsAt 'date-time-tz-long' tz=this.event.timezone}}
-          {{/if}}
+          <Public::EventDateTime @event={{@event}} />
```

Comment:
why removed `if` ? why removed dates (correct me if it was a requirement of issue)?

Reply:
@maze-runnar I was told to make changes in event card  to remove code duplicacy, please see above comment .Dates and location have been replaced by components in both order-card and event-card.